src/webgl/interaction.js

```diff
@@ -22,6 +22,7 @@ import * as constants from '../core/constants';
  * @for p5
  * @param  {Number} [sensitivityX] sensitivity to mouse movement along X axis
  * @param  {Number} [sensitivityY] sensitivity to mouse movement along Y axis
+ * @param  {Number} [sensitivityZoom] sensitivity to scroll movement along Z axis
```

Comment:
curious why this is named `sensitivityZoom` and not `sensitivityZ`?

Reply:
Thanks for catching that. I was unsure of whether to use a more descriptive term for the sensation that happens. A lot of the time in 3D software the mouse scroll movement will be described as the 'zoom'. That said, I think that I shouldn't break with the other two parameters and  I should make it sensitivityZ. People will likely be able to connect the sensation of zooming with z movement. Making changes now!